src/main/java/org/jenkinsci/plugins/workflow/support/storage/BulkFlowNodeStorage.java

```diff
@@ -234,6 +229,13 @@ private void storeActions() {  // We've already loaded the actions, may as well
     private static final Method FlowNode_setActions;
 
     static {
+        // Aliases reduce the amount of data persisted to disk
+        XSTREAM.alias("Tag", Tag.class);
```

Comment:
This will render older versions of the plugin unable to read builds created by newer versions. That is not inherently a problem, but deserves a [`compatibleSinceVersion` note in the update center](http://jenkinsci.github.io/maven-hpi-plugin/jar-mojo.html#compatibleSinceVersion).

Reply:
Yep, it'll get some sort of note -- the Changlog is also a good place. 

Worth it because this reduces the amount of FlowNode data written quite significantly (30% or more).  For performance, this isn't as important as cutting the number of IO calls made, but it also reduces size-on-disk for the flowgraph.